vars/sectionNightlyTests.groovy

```diff
@@ -49,6 +49,52 @@ def call(PipelineCallbacksRunner pcr, AppPipelineConfig config, PipelineType pip
         }
       }
     }
+    if (config.parallelCrossBrowserTest) {
```

Comment:
why not just re-use the existing one? best to give everyone the benefit?

probably worth the extra testing?

any blockers for using the existing one?

Reply:
I don't think all the teams have implemented the crossbrowser tests in the exact same way, and I'm pretty sure they don't have separate scripts in package.json to support
```
yarn test:crossbrowser-chrome
yarn test:crossbrowser-firefox
...
```
But that's an interesting thought, as a few teams already support using a `BROWSER_GROUP` environment variable to define the browser to run, so maybe that's a better route to go 🤔. I'll have a look at a few services to see how common that approach is and maybe move towards that instead

Reply:
But even still, I don't think we'd be able to update the current functionality without breaking alot of service's current tests.

I can add a `@deprecated` flag to the old method once I've tested the new one fully

Reply:
sure, a standard `BROWSER` env flag would be good

Reply:
I've made those changes now, using the `BROWSER_GROUP` env var to set the browser type, as this is already a commonly used variable for crossbrowser testing in service teams. It also reduces any potential impact from an env var called BROWSER already being used for something else.